PhysicsTools/TensorFlow/plugins/TfGraphDefProducer.cc

```diff
@@ -21,7 +21,7 @@
 
 #include "FWCore/Framework/interface/ESHandle.h"
 #include "TrackingTools/Records/interface/TfGraphRecord.h"
```

Comment:
should this file be migrated as well? (should be generic enough)

https://github.com/cms-sw/cmssw/blob/6d2f66057131baacc2fcbdd203588c41c885b42c/TrackingTools/Records/interface/TfGraphRecord.h#L1-L22

(despite the description).
If so what's a good package to move it?

Reply:
good point. by analogy with MVAComputer, is there any objection to putting the TfGraphRecord in PhysicsTools/TensorFlow?

```
PhysicsTools/MVAComputer/interface/MVAComputerRecord.h:#include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
```

Reply:
so I'll go ahead with this change as well, but I'll wait for tomorrow to see if there are further comments from the ML people.

Reply:
done at [ae0c89c](https://github.com/cms-sw/cmssw/pull/37968/commits/ae0c89c43bb3110521519bf1d19009457dbe0eac)